libs/components/src/lib/send-offer-modal/send-offer-modal.tsx

```diff
@@ -89,55 +99,53 @@ export function SendOfferModal(props: Props) {
             {...register('rate', { required: true })}
             name="rate"
             control={control}
-            render={({ field }) => (
+            render={({ field: { onChange, name, value } }) => (
               <Row justify="start">
                 <Col span={8}>
                   <p>{t('modalInvite.rate')}</p>
                 </Col>
                 <Col span={6}>
-                  <Form.Item {...field}>
-                    <Input
-                      defaultValue={rate}
-                      type="number"
-                      placeholder={t('modalInvite.placeholder')}
-                      {...field}
-                    />
-                  </Form.Item>
+                  <Input
+                    onChange={e => {
+                      onChange(parseInt(e.target.value));
+                    }}
+                    defaultValue={rate}
+                    type="number"
+                    placeholder={t('modalInvite.placeholder')}
+                  />
                 </Col>
               </Row>
             )}
           />
 
           <Controller
-            {...register('start', { required: true })}
-            name="start"
             control={control}
-            render={({ field }) => (
+            name="start"
+            render={({ field: { onChange, name, value } }) => (
               <Row justify="start">
                 <Col span={8}>
                   <p>{t('modalInvite.time')}</p>
                 </Col>
                 <Col span={6}>
-                  <Form.Item {...field}>
-                    <DatePicker
-                      defaultValue={dayjs(todayDate, dateType)}
-                      picker="date"
-                    />
-                  </Form.Item>
+                  <DatePicker
+                    onChange={date => {
+                      onChange(date?.isValid ? date : '');
+                    }}
+                    format={'MM/DD/YYYY'}
```

Comment:
format should be one for full project, so it should be global constant

Reply:
fixed